docs/ENVIRONMENT.md

```diff
@@ -105,6 +105,13 @@ systemd-timedated:
   first existing unit listed in the environment variable, and
   `timedatectl set-ntp off` disables and stops all listed units.
 
+bootctl and other tools that access the EFI System Partition (ESP):
+
+* `$SYSTEMD_RELAX_ESP=1` — if set, the ESP validation checks are
```

Comment:
Hmm, not crazy about the name. Maybe `SYSTEMD_ESP_CHECKS=0` instead? Or even `SYSTEMD_RELAX_ESP_CHECKS=1` ?